resources/js/Pages/Airports/AirportDetail.js

```diff
@@ -21,6 +21,19 @@ const AirportDetail = ({ airport, metar, aircraft }) => {
     }
   }
 
+  const renderAirportSize = (size) => {
```

Comment:
I think this needs some thinking, as it makes my local small airfield a Major airport. Maybe if we leave out for now, and then come back to.

Reply:
Happy to leave out the declaration and invocation of renderAirportSize for now. Can you do that?

Reply:
Hey sorry - just seen this. If you checkout your pr branch remove it, commit and push up it will update your PR

Reply:
Have removed just the renderAirporSize for now (removed call, commended the declaration)